skimage/measure/segmentation_metrics.py

```diff
@@ -0,0 +1,493 @@
+import numpy as np
+import multiprocessing
+from .simple_metrics import _assert_compatible
+import scipy.sparse as sparse
+from scipy.ndimage.measurements import label
+
+__all__ = [ 'compare_adapted_rand_error',
+            'compare_rand_by_threshold',
+            'compare_raw_edit_distance',
+            'compare_split_vi',
+            'compare_vi',
+          ]
+
+def compare_raw_edit_distance(im_true, im_test, size_threshold=1000):
+    """Compute the edit distance between two segmentations.
+
+    Parameters
+    ----------
+    im_true : ndarray
+        Ground-truth image.
+    im_test : ndarray
+        Test image.
+    size_threshold : int or float, optional
+        Ignore splits or merges smaller than this number of voxels.
+
+    Returns
+    -------
+    (false_merges, false_splits) : float
+        The number of splits and merges required to convert aseg to gt.
+    """
+    _assert_compatible(im_true, im_test)
+
+    im_test = relabel_from_one(im_test)[0]
+    im_true = relabel_from_one(im_true)[0]
```

Comment:
This is now `morphology.relabel_sequential` in scikit-image

Reply:
>>> import skimage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\scikitimage\scikit-image\skimage\__init__.py", line 137, in <module>
    from .data import data_dir
  File "d:\scikitimage\scikit-image\skimage\data\__init__.py", line 16, in <module>
    from ._binary_blobs import binary_blobs
  File "d:\scikitimage\scikit-image\skimage\data\_binary_blobs.py", line 2, in <module>
    from ..filters import gaussian
  File "d:\scikitimage\scikit-image\skimage\filters\__init__.py", line 3, in <module>
    from .edges import (sobel, sobel_h, sobel_v,
  File "d:\scikitimage\scikit-image\skimage\filters\edges.py", line 17, in <module>
    from ..restoration.uft import laplacian
  File "d:\scikitimage\scikit-image\skimage\restoration\__init__.py", line 11, in <module>
    from .inpaint import inpaint_biharmonic
  File "d:\scikitimage\scikit-image\skimage\restoration\inpaint.py", line 8, in <module>
    from ..measure import label
  File "d:\scikitimage\scikit-image\skimage\measure\__init__.py", line 8, in <module>
    from .segmentation_metrics import (compare_adapted_rand_error,
  File "d:\scikitimage\scikit-image\skimage\measure\segmentation_metrics.py", line 6, in <module>
    **from ..segmentation import relabel_sequential**
  File "d:\scikitimage\scikit-image\skimage\segmentation\__init__.py", line 1, in <module>
    from .random_walker_segmentation import random_walker
  File "d:\scikitimage\scikit-image\skimage\segmentation\random_walker_segmentation.py", line 43, in <module>
    from ..filters import rank_order
ImportError: cannot import name 'rank_order'


I think changing this created a circular dependency, any suggestions?